packages/components/src/menu/src/Menu.css

```diff
@@ -88,16 +88,20 @@ IMPORTANT: The Menu component hardcoded a few CSS values to enable dynamic scrol
     background-color: var(--o-ui-bg-alias-mid-break);
 }
 
-/* OPTION | STATE | FOCUS | WITH SELECTION */
-.o-ui-menu-with-selection .o-ui-menu-item:focus:after,
-.o-ui-menu-with-selection .o-ui-menu-item-focus:after {
+/* FOCUS */
+.o-ui-menu-item::after {
     content: "";
     display: block;
     position: absolute;
     left: 0;
-    height: var(--o-ui-menu-item-height);
+    height: 100%;
```

Comment:
Pretty sure the item height is to support a scrollbar in a menu. Did you try test it?

Reply:
All is good chromatic wise

Reply:
I am not sure there is a Chromatic test for this for a Menu. You should verify.

Reply:
![image](https://user-images.githubusercontent.com/361632/152049546-fb072c0f-9a90-449c-a6f0-9f7ba9290c0f.png)

I did a check too.